### DISCUSSION TITLE
Does TerraOS wipe your Chromebook when you boot back into Chrome OS?

### DISCUSSION THREAD

**Discussion by leoisbadatmakeingthings** [3kh0/ext-remover#1096] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I saw a video on it and I really want to try it but when they booted back into Chrome OS it wiped their Chromebook. I really don't want to wipe my Chromebook. Is there a way to get back into Chrome OS without wiping your Chromebook.

**Comment by CGRmags** [comment]
you know.......I dont know

**Comment by FunnyCheese4Me** [comment]
Can you link the video?

**Reply by leoisbadatmakeingthings** [reply]
https://www.youtube.com/watch?v=2YPWLPbnjg4
the only video i found about it on google

**Reply by FunnyCheese4Me** [reply]
I'll check it out.

**Comment by FunnyCheese4Me** [comment, marked as answer]
Yeah, it does. There's no way to avoid it, but you might be able to find and get your local storage mounted in arch or something. From there you can back up your files if you can find your user folder and access it in read-only(I believe it won't let you have write because user `terraos` doesn't own it), and back stuff up.
Also I had a thought: building terraos... in terraos.
Imagine.
